### DISCUSSION TITLE
Is anybody else's server-side custom CSS not working in JMP in the latest version?

### DISCUSSION THREAD

**Discussion by TheHYPO35** [jellyfin/jellyfin-desktop#695] [General]
I have some custom CSS on my server that appears to be working in the browser version, but isn't working in JMP at least on the latest version (I don't think this was an issue on the version before this one)?

**Comment by Druidblack** [comment]
The player ignores such css code.

body:not(.hide-scroll) > .videoPlayerContainer > video {
   object-fit: cover;
}

https://github.com/EusthEnoptEron/jellyfin-plugin-animethemes?tab=readme-ov-file#other-notes

 Because of this, the thematic video is displayed in full screen.
The setting should stretch the video of any resolution to the full screen (to hide the black bars). And JMP ignores this and shows the background video as it is.
![bag](https://github.com/user-attachments/assets/32d2cf68-1933-4f47-9219-0f4b822a6c47)
![cool](https://github.com/user-attachments/assets/5aabb7cd-e2b9-434c-b947-87424f4fc114)

**Comment by Druidblack** [comment]
At the same time, the css of the visual theme is applied correctly
